pallets/oracle/src/lib.rs

```diff
@@ -281,16 +253,26 @@ impl<T: Config> Pallet<T> {
 		}
 	}
 
+	// TODO
 	// public only for testing purposes
-	pub fn _feed_values(oracle: T::AccountId, values: Vec<(OracleKey, T::UnsignedFixedPoint)>) {
-		for (key, value) in values.iter() {
-			let timestamped =
-				TimestampedValue { timestamp: Self::get_current_time(), value: *value };
-			RawValues::<T>::insert(key, &oracle, timestamped);
-			RawValuesUpdated::<T>::insert(key, true);
+	#[cfg(feature = "testing-utils")]
+	pub fn _feed_values(
+		oracle: T::AccountId,
```

Comment:
Since the introduction of this `oracle` parameter to this function results in lots of (actually unnecessary) changes to the existing code, since all the calls to `_feed_values` have to be adjusted, I would suggest we remove this parameter again and in l. 267, when calling 
```
T::DataFeedProvider::feed_value(oracle.clone(), k.clone(), timestamped)
				.expect("Expect store value by key");
```
we just use some hardcoded account ID. Since this is only for testing purposes I don't see a problem with always using the same account for it. We can just do something like
```
let feeder = AccoundId::from([1u8;32])  
T::DataFeedProvider::feed_value(feeder, k.clone(), timestamped)
				.expect("Expect store value by key");
```

Reply:
Thanks. Good suggestion. 
I tried this approach to mock account id. for this testing function.
But it does not work

```
let feeder = T::AccountId::from([1u8; 32]);
    |                      ------------------ ^^^^^^^^^ expected associated type, found array `[u8; 32]`
    |                      |
    |                      arguments to this function are incorrect
    |
    = note: expected associated type `<T as frame_system::Config>::AccountId`
                         found array `[u8; 32]`
```

But this is really works. Thanks.

```
use primitives::AccountId;
let feeder = AccountId::from([1u8; 32]);
```

Reply:
I assume `T::AccountId::new(Default::default())` also does not work?

Reply:
It also does work. @ebma 

```
let r = T::AccountId::new(Default::default());
    |                               ^^^ function or associated item not found in `<T as frame_system::Config>::AccountId`
```

```
let r = T::AccountId::from(Default::default());
    |                                    ^^^^^^^^^^^^^^^^ the trait `std::default::Default` is not implemented for `<T as frame_system::Config>::AccountId`
```

Reply:
Okay, then I'm fine with keeping it the way you implemented it.